### PR TITLE
Add Artifactory example

### PR DESCRIPTION
**Pull request by ssk2**
Continuing on from #52.

### CODE REVIEW

**Comment by ssk2** [comment]
This needs to be updated to reflect the new way to install the latest version of the Artifactory package in https://github.com/mesosphere/universe/pull/1007

**Comment by ssk2** [comment]
TODO: 
- [ ] Clean up "using Artifactory" and add details on how to push Docker images from Jenkins (or a local machine).
- [ ] Update Artifactory Enterprise instructions to reflect current state of package.
- [ ] Move MySQL instructions to the right place and tidy these up.

File: 1.8/artifactory/artifactory-enterprise.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      "enabled": false,
+      "unique-nodes": true,
+      "nodes": 1,
+      "name": "artifactory"
+    }
+  }
+}
+```
+
+ ###NOTE: Make sure database name, is correct in connection-string as well as username & password for database.
```
Comment (colin-msphere) [inline 1.8/artifactory/artifactory-enterprise.md]: Can remove the comma after name (`name, `).

**Comment by joerg84** [comment]
@ssk2 did you note the new folder structure?

**Comment by ssk2** [comment]
I did not.

**Pull request merged**